pages/wiki/building-asteroidos.hbs

```diff
@@ -10,35 +10,91 @@ layout: documentation
-
-<pre><code>git config --global user.email "you@example.com"
-git config --global user.name "Your Name"</code></pre>
-
 <p>Clone the main repository using the following command:</p>
 <pre><code>git clone https://github.com/AsteroidOS/asteroid.git
 cd asteroid/
 </code></pre>
-<p>This repository only contains a shell script that populates <em>src/</em> with OpenEmbedded and the appropriate Asteroid layers. Then, it setups the environment for a bitbake build. The following command will setup a build for <em>dory</em> (the LG G Watch) but you can also build an image for other watches by using the corresponding codename.</em></p>
+
+<p>If you haven't set a global git name and email yet, please do so, otherwise the build will not succeed! For example:</p>
+
+<pre><code>git config --global user.email "you@example.com"
+git config --global user.name "Your Name"</code></pre>
+
+<p>This repository basically only contains a shell script that populates <em>src/</em> with OpenEmbedded and the appropriate Asteroid layers. Then, it setups the environment for a bitbake build. The following command will setup a build for <em>dory</em> (the LG G Watch) but you can also build an image for other watches by using the corresponding codename. (Codenames can be found on the <a href="{{rel 'install'}}">Install page</a>.)</p>
 <pre><code>. ./prepare-build.sh dory # Be careful that this script must be sourced and not only ran
 </code></pre>
-<p>If the environment has been correctly setup, you should now be in the <i>build</i> subdirectory.</p>
+<p>If the environment has been correctly setup, you should now be in the <em>build</em> subdirectory.</p>
 <div class="page-header">
-  <h1 id="building">Building</h1>
+  <h2 id="building">Building</h2>
 </div>
 <p>Once the environment is prepared, you can simply trigger a build with the following command:</p>
 <pre><code>bitbake asteroid-image</code></pre>
-<p><i>Note:</i> Bitbake is a powerful tool that can also build single packages (e.g: bitbake strace) or <a href="{{rel 'wiki/building-asteroidos'}}">the SDK</a> for example. Refer to its documentation for more details.</p>
+<p><em>Note:</em> Bitbake is a powerful tool that can also build single packages (e.g: bitbake strace) or <a href="{{rel 'wiki/building-asteroidos'}}">the SDK</a> for example. Refer to its documentation for more details.</p>
+
+
+<div class="page-header">
+  <h1 id="build-with-docker">Build with Docker</h1>
+</div>
+
+<div class="page-header">
+  <h2 id="docker-build-setup">Setup</h2>
+</div>
+
+<p>First install Docker and Git on your system if you haven't already, then clone the main repository using the following command:</p>
+<pre><code>git clone https://github.com/AsteroidOS/asteroid.git
+cd asteroid/</code></pre>
+
+<p>If you haven't set a global git name and email yet, please do so, otherwise the build will not succeed! For example:</p>
```

Comment:
the "please do so, otherwise the build will not succeed!" is not a wording that should be found in a documentation. You should keep a neutral tone and it is implied that the instructions you give are necessary for the build to succeed, otherwise you wouldn't give them on the first place.
I would suggest "If you haven't set a global git name and email yet, adapt the following commands to your information." If you want you can also justify this with something like "This is only required to clone some git repositories when building" but no exclamation mark is necessary.